search/mindbender-search

```diff
@@ -157,14 +161,15 @@ case $Action in
             local relation=$1; shift
             local sqlToUnload=$1; shift
             local jqToFormat=$1; shift
+            local length_relation="$(deepdive sql eval "SELECT COUNT(*) FROM $relation")"
             echo >&2 "Indexing relation $relation"
             # unload data from database in json lines
             deepdive sql eval "$sqlToUnload" format=json |
             # show progress
-            pv --line-mode --size="$(deepdive sql eval "SELECT COUNT(*) FROM $relation")" |
+            pv --line-mode --size=$length_relation |
             # split records into multiple batches
             # TODO parallelize
-            split --lines=$ELASTICSEARCH_BULK_BATCHSIZE --filter="$(escape-args-for-shell sh -euc '
+            split --lines=$ELASTICSEARCH_BULK_BATCHSIZE --numeric-suffixes --suffix-length=$(expr $(($length_relation / $ELASTICSEARCH_BULK_BATCHSIZE +1)) : '.*') --filter="$(escape-args-for-shell sh -euc '
```

Comment:
Why do you need the `expr`?  I think it's unnecessary: `--suffix-length=$(($length_relation / $ELASTICSEARCH_BULK_BATCHSIZE +1))`